src/components/task-plan/footer.cjsx

```diff
@@ -47,8 +55,8 @@ PlanFooter = React.createClass
         <AsyncButton
           bsStyle='primary'
           className='-publish'
-          onClick={onPublish}
-          isWaiting={isWaiting}
+          onClick={@onPublish}
+          isWaiting={isWaiting and @state.publishing}
```

Comment:
can you use `isSaving` instead of `saving` on `@state`? That way you can do:

`{isSaving, isPublishing} = @state` and then use it below